applications/app/views/fragments/galleryHeader.scala.html

```diff
@@ -3,23 +3,38 @@
 @import views.support.TrailCssClasses.toneClass
 
 <header class="content__head tonal__head tonal__head--@toneClass(gallery.item)">
-    @* Hidden because we visually show this data in the head, but needed
-    here for SEO. *@
+    @* Hidden because we visually show this data in the head, but needed here for SEO. *@
     <h1 class="is-hidden" itemprop="headline">@Html(gallery.item.trail.headline)</h1>
 
     <div class="gs-container">
-        @fragments.commercial.badge(gallery.item, gallery)
+        <div class="gallery__header">
 
-        @fragments.contentMeta(gallery.item, gallery)
+            <div class="gallery__badge--container">
+                @fragments.commercial.badge(gallery.item, gallery)
+            </div>
 
-        @if(gallery.item.fields.standfirst.isDefined) {
-            <div class="tonal__standfirst">
-                @fragments.standfirst(gallery.item)
+            <div class="meta__extras--gallery">
```

Comment:
> Normally we would have something like meta__extras meta__extras--gallery and this pattern is one of the things which has caused overrides.

I agree and think, if we would only use midifiers on the top-level component we'd have much less confusion about it.

I am not sure if this should even have a similar name, maybe something like gallery__social-container would be better?

> I am not sure if this should even have a similar name, maybe something like gallery__social-container would be better?

Agreed. What do you think about `gallery_meta-extras`? Otherwise I'd say `gallery .meta__extras` could also be fine.